ext/node/resolution.rs

```diff
@@ -329,46 +325,30 @@ impl NodeResolver {
     Ok(resolved)
   }
 
-  pub fn resolve_npm_req_reference(
-    &self,
-    reference: &NpmPackageReqReference,
-    mode: NodeResolutionMode,
-    permissions: &dyn NodePermissions,
-  ) -> Result<Option<NodeResolution>, AnyError> {
-    let pkg_id = self
-      .npm_resolver
-      .resolve_pkg_id_from_pkg_req(reference.req())?;
-    let reference = NpmPackageNvReference::new(PackageNvReference {
-      nv: pkg_id.nv,
-      sub_path: reference.sub_path().map(ToOwned::to_owned),
-    });
-    self.resolve_npm_reference(&reference, mode, permissions)
-  }
-
   pub fn resolve_npm_reference(
     &self,
-    reference: &NpmPackageNvReference,
+    package_folder: &Path,
+    sub_path: Option<&str>,
     mode: NodeResolutionMode,
     permissions: &dyn NodePermissions,
   ) -> Result<Option<NodeResolution>, AnyError> {
-    let package_folder = self
-      .npm_resolver
-      .resolve_package_folder_from_deno_module(reference.nv())?;
     let node_module_kind = NodeModuleKind::Esm;
     let maybe_resolved_path = self
       .package_config_resolve(
-        &reference
-          .sub_path()
+        &sub_path
           .map(|s| format!("./{s}"))
           .unwrap_or_else(|| ".".to_string()),
-        &package_folder,
+        package_folder,
         node_module_kind,
         DEFAULT_CONDITIONS,
         mode,
         permissions,
       )
       .with_context(|| {
-        format!("Failed resolving package config for '{reference}'")
+        format!(
+          "Failed resolving package config for '{}'",
+          package_folder.join("package.json").display()
```

Comment:
Maybe update this error? "Package config" suggests `package.json` file so using `package_folder.join("package.json")` seems superfluous

Reply:
What would you recommend?

Reply:
`Failed resolving {package_json_path}` or keep the existing error.

Reply:
I think that makes it sound like it can't find the filepath, which might not be the case.